Bio/TogoWS/__init__.py

```diff
@@ -189,9 +189,10 @@ def search_iter(db, query, limit=None, batch=100):
        TogoWS (currently limited to 100).
 
     You would use this function within a for loop, e.g.
-
-    >>> for id in search_iter("pubmed", "lung+cancer+drug", limit=10):
-    ...     print(id) # maybe fetch data with entry?
+    >>> from Bio import TogoWS
+    >>> for id in TogoWS.search_iter("pubmed", "diabetes+human", limit=10):
+    ...     print("PubMed ID: %s" %id) # maybe fetch data with entry?
+    PubMed ID: 27374092
 
```

Comment:
You need to replace the id (27374092) with an ELLIPSIS (...) because the search result (and thus the first id) will change when new articles with this keywords will be added to Pubmed. The ELLIPSIS (three dots: ...) will match any output.

Reply:
Amended, Thank You.